.github/workflows/ci.yml

```diff
@@ -38,6 +51,16 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ env.NODE_VERSION }}
+      - name: Cache node_modules
+        id: cache-node-modules
+        uses: actions/cache@v2
+        with:
+          path: | 
+            node_modules
+            projects/storefrontapp-e2e-cypress/node_modules
```

Comment:
same as other comment, should we try to install the cypress package as well in case? (library versions might differ)